D4Group.cc

```diff
@@ -162,6 +162,22 @@ D4Group *D4Group::find_child_grp(const string &grp_name) {
     return (g == grp_end()) ? 0 : *g;
 }
 
+D4Group *D4Group::find_grp_internal(const string &grp_path) {
```

Comment:
Since the path lookup code can be kind of complicated, let's add a comment here that this is a private method used used for relative paths only. Unless that's not true of course ;-)